content/chat/rooms/messages.textile

```diff
@@ -97,6 +108,13 @@ blang[javascript].
 blang[react].
   When you unmount the component that is using the @useMessages@ hook, it will automatically handle unsubscribing any associated listeners registered to receive messages.
 
+blang[swift].
+  To cancel the room messages subscribtion, call the provided @finish()@ method:
```

Comment:
General question, is `finish` idiomatic in Swift? In JS we preferred `unsubscribe` instead.

Reply:
updated